.github/workflows/relase.yml

```diff
@@ -0,0 +1,41 @@
+name: Release
+
+on:
+  pull_request_target:
+    branches:
+      - 'release/v[1-9]+.[0-9]+.[0-9]+'
+    types:
+      - closed
+
+jobs:
+  release:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get branch name
+        id: branch-name
+        uses: tj-actions/branch-names@v5.2
+      - uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '14.x'
+          registry-url: https://registry.npmjs.org/
+      - name: Install dependencies
+        run: yarn install
+      - name: Prettier
+        run: yarn prettier-check
+      - name: Lint
+        run: yarn lint
+      - name: Combine
+        run: yarn combine
+      - name: Compile
+        run: yarn compile
+      - name: Build
+        run: yarn build
+        env:
+          NODE_OPTIONS: --max-old-space-size=8192
```

Comment:
This virtual machines don't have 8GB memory at all. 😄 
<img width="636" alt="image" src="https://user-images.githubusercontent.com/3479963/163568791-013ea18a-e7d5-4843-b220-721701e599e2.png">
Maybe we can try OSX as well, if it is free maybe a little bit faster than ubuntu.

Reply:
That's a nice catch, updated to 7GB.